### PR TITLE
[backend] store enum filters values in constants

### PR DESCRIPTION
**Pull request by Archidoit**
### Proposed changes
Enum filters values where hardwritten in registration attributes files. 
Now we refer to constants to avoid missing values in filters, as it was the case in https://github.com/OpenCTI-Platform/opencti/issues/8414

### Related issues
avoid issues like https://github.com/OpenCTI-Platform/opencti/issues/8414

### CODE REVIEW

File: opencti-platform/opencti-front/lang/back/fr.json [hunk 1]
```diff
@@ -192,6 +192,7 @@
   "Description": "Description",
   "Descriptions": "Descriptions",
   "Detection": "Détection",
+  "Digest period": "Période de digestion",
```
Comment (marieflorescontact) [inline opencti-platform/opencti-front/lang/back/fr.json]: this is weird : a digest is more like "un résumé/condensé"

File: opencti-platform/opencti-graphql/src/modules/internal/document/document.ts [hunk 2]
```diff
@@ -17,7 +18,7 @@ const attributes: Array<AttributeDefinition> = [
   { name: 'information', label: 'Information', type: 'string', format: 'short', mandatoryType: 'internal', editDefault: false, multiple: false, upsert: false, isFilterable: false },
   { name: 'lastModified', label: 'Last modification date', type: 'date', mandatoryType: 'internal', editDefault: false, multiple: false, upsert: false, isFilterable: true },
   { name: 'lastModifiedSinceMin', label: 'Last modification since', type: 'numeric', precision: 'integer', mandatoryType: 'internal', editDefault: false, multiple: false, upsert: false, isFilterable: false },
-  { name: 'uploadStatus', label: 'Upload status', type: 'string', format: 'enum', values: ['wait', 'progress', 'complete', 'timeout'], mandatoryType: 'internal', editDefault: false, multiple: false, upsert: false, isFilterable: true },
+  { name: 'uploadStatus', label: 'Upload status', type: 'string', format: 'enum', values: Object.values(State), mandatoryType: 'internal', editDefault: false, multiple: false, upsert: false, isFilterable: true },
```
Comment (labo-flg) [inline opencti-platform/opencti-graphql/src/modules/internal/document/document.ts]: for consistency, could you please put it as a constant above ? like in other files.

File: opencti-platform/opencti-front/lang/back/fr.json (hunk 1 above)
Comment (Archidoit) [reply opencti-platform/opencti-front/lang/back/fr.json]: en effet :D

File: opencti-platform/opencti-graphql/src/modules/internal/document/document.ts (hunk 2 above)
Comment (Archidoit) [reply opencti-platform/opencti-graphql/src/modules/internal/document/document.ts]: fixed

**Review by labo-flg — APPROVED** [review]

**Pull request merged**